### ISSUE TITLE
Star System config not working.

### ISSUE THREAD

**Issue by Ciborgm9** [xen-42/outer-wilds-new-horizons#103]
### What Happened?

The configurations in my Star System schema dont work.  

### Platform

Steam

### Mods

```Markdown
PacificEngine's Common Resources, New Horizons and Menu Framework
```

**Comment by Ciborgm9** [comment]
Update: the "canEnterViaWarpDrive" config seems to work, the problem is with "startHere"

**Comment by xen-42** [comment]
Fixed in v0.16.0
